docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -41,7 +41,15 @@ If nothing obvious from the above, check for increased load:
 
 ### MimirIngesterReachingSeriesLimit
 
-This alert fires when the `max_series` per ingester instance limit is enabled and the actual number of in-memory series in an ingester is reaching the limit. Once the limit is reached, writes to the ingester will fail (5xx) for new series, while appending samples to existing ones will continue to succeed.
+This alert fires when the `max_series` per ingester instance limit is enabled and the actual number of in-memory series in an ingester is reaching the limit.
+The threshold is set at 80%, to give some chance to react before the limit is reached.
+Once the limit is reached, writes to the ingester will fail for new series. Appending samples to existing ones will continue to succeed.
```

Comment:
```suggestion
After the limit is reached, writes to the ingester fail for new series. Appending samples to existing ones continue to succeed.
```

Reply:
Can we avoid using "writes" as a noun? Could we say "write requests" or something else, if it's more accurate. We could then also say "Appending samples to existing requests continue to succeed" for greater clarity.

Reply:
We avoid using "will" in the docs.